src/styles/dotcms-theme/components/_treetable.scss

```diff
@@ -1,203 +1,251 @@
+    background: rgba(63, 81, 181, 0.12);
+    margin-left: $spacing-1;
 }
+
 .p-treetable .p-sortable-column:not(.p-highlight):hover {
-  background: rgba(0, 0, 0, 0.04);
-  color: $black;
+    background: $hover-bg;
+    color: $hover-text-color;
 }
+
 .p-treetable .p-sortable-column:not(.p-highlight):hover .p-sortable-column-icon {
-  color: rgba(0, 0, 0, 0.6);
+    color: $hover-text-color;
 }
+
 .p-treetable .p-sortable-column.p-highlight {
-  background: $white;
-  color: $black;
+    background: $white;
+    color: $black;
 }
+
 .p-treetable .p-sortable-column.p-highlight .p-sortable-column-icon {
-  color: $black;
+    color: $black;
 }
+
 .p-treetable .p-treetable-tbody > tr {
-  background: $white;
-  color: $black;
-  transition: none;
-  outline-color: transparent;
+    background: $white;
+    color: $black;
+    transition: none;
+    outline-color: transparent;
 }
+
 .p-treetable .p-treetable-tbody > tr > td {
-  text-align: left;
-  border: 1px solid #e4e4e4;
-  border-width: 0 0 1px 0;
-  padding: $spacing-2 $spacing-2;
+    text-align: left;
+    border: 1px solid #e4e4e4;
+    border-width: 0 0 1px 0;
+    padding: $spacing-2 $spacing-2;
 }
+
 .p-treetable .p-treetable-tbody > tr > td .p-treetable-toggler {
-  width: $spacing-5;
-  height: $spacing-5;
-  color: rgba(0, 0, 0, 0.6);
-  border: 0 none;
-  background: transparent;
-  border-radius: 50%;
-  transition: background-color 0.2s, color 0.2s, box-shadow 0.2s;
-  margin-right: $spacing-1;
+    width: $spacing-5;
+    height: $spacing-5;
+    color: rgba(0, 0, 0, 0.6);
+    border: 0 none;
+    background: transparent;
+    border-radius: 50%;
+    transition: background-color 0.2s, color 0.2s, box-shadow 0.2s;
+    margin-right: $spacing-1;
 }
+
 .p-treetable .p-treetable-tbody > tr > td .p-treetable-toggler:enabled:hover {
-  color: rgba(0, 0, 0, 0.6);
-  border-color: transparent;
-  background: rgba(0, 0, 0, 0.04);
+    color: $hover-text-color;
+    border-color: transparent;
+    background: $hover-bg;
 }
+
 .p-treetable .p-treetable-tbody > tr > td .p-treetable-toggler:focus {
-  outline: 0 none;
-  outline-offset: 0;
-  box-shadow: none;
+    outline: 0 none;
+    outline-offset: 0;
+    box-shadow: none;
 }
+
 .p-treetable .p-treetable-tbody > tr > td p-treetablecheckbox .p-checkbox {
-  margin-right: $spacing-1;
+    margin-right: $spacing-1;
 }
+
 .p-treetable .p-treetable-tbody > tr > td p-treetablecheckbox .p-checkbox .p-indeterminate .p-checkbox-icon {
-  color: $black;
+    color: $black;
 }
+
 .p-treetable .p-treetable-tbody > tr.p-highlight {
-  background: $highlight-bg;
-  color:  $highlight-text-color;
+    background: $highlight-bg;
+    color: $highlight-text-color;
 }
+
 .p-treetable .p-treetable-tbody > tr.p-highlight .p-treetable-toggler {
-  color:  $brand-primary;
+    color: $brand-primary;
 }
+
 .p-treetable .p-treetable-tbody > tr.p-highlight .p-treetable-toggler:hover {
-  color:  $brand-primary;
+    color: $hover-text-primary-color;
```

Comment:
not sure about the name `$hover-text-primary-color` need to see other uses for this one.